src/foam/u2/wizard/wizardlet/SuccessWizardletView.js

```diff
@@ -16,7 +16,7 @@ foam.CLASS({
       flex-direction: column;
       justify-content: center;
       align-items: center;
-
+      margin-bottom: 50px;
```

Comment:
```suggestion
     padding: 2rem 0;
```
Padding is preferred over margins since margins can collapse